pyhgvs/__init__.py

```diff
@@ -970,6 +970,13 @@ def parse_genome(self, details):
 
         raise InvalidHGVSName(details, 'genomic allele')
 
+    def _validate(self):
+        """
+        Check for internal inconsistencies in representation
+        """
+        if self.start > self.end:
```

Comment:
I didn't find any place in the [hgvs spec](http://www.hgvs.org/mutnomen/recs-DNA.html#del) that disallows having the same start and stop, like `78_78delC`, so I think that this is fine as is. `78delC` might be a preferred way to encode such a change.